spec/helpers/stylesheet.js

```diff
@@ -37,6 +37,35 @@ describe('stylesheet helper', () => {
         });
     });
 
+    it('should render both link tags correclty but produce only one resource hint', done => {
```

Comment:
This test exemplifies what's described in this comment https://github.com/bigcommerce/paper-handlebars/pull/212/files#r998713714